charts/opentelemetry-operator/values.yaml

```diff
@@ -186,6 +186,12 @@ admissionWebhooks:
   failurePolicy: Fail
   secretName: ""
 
+  ## Defines the sidecar injection logic in Pods.
+  ## - Ignore, the injection is fail-open. The pod will be created, but the sidecar won't be injected.
+  ## - Fail, the injection is fail-close. If the webhook pod is not ready, pods cannot be created.
+  pods:
```

Comment:
@dacamposol please update the json schema file.

Reply:
Done, but just for information, the JSON Schema isn't following your own defined configuration.

From what I see, the indentation is of 4 spaces in the existing file, but afterwards in `.editorconfig` you have defined the indent to be 2:

```.editorconfig
[*.{json,yaml}]
indent_style = space
indent_size = 2
```

Are you maybe autogenerating that file and not formatting it afterwards?

Reply:
Yes I think it was being auto-generated for a time.